docs/articles/documentation/using-testcafe/command-line-interface.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: docs
+title: Command Line Interface
+permalink: /documentation/using-testcafe/command-line-interface/
+---
+# Command Line Interface
+
+TestCafe command line interface is pretty simple.
+It consists of a single command that runs tests with a number of options.
+
+#### Synopsis
+
+```sh
+testcafe [options] <browser-list-comma-separated> <file-or-glob ...>
+```
+
+#### Browser List
+
+Specify the list of browsers where tests will be ran.
+You can specify a browser alias or path.
+
+To run tests against all installed browsers, use the *all* alias.
+
+To use remote browser connection (to run tests on a remote device),
+specify "remote" as a browser alias. If you want to connect multiple devices,
+prefix alias with the number of browsers you want to connect (e.g. "3remote").
```

Comment:
Need to split this into 3 different topics:
1. Local browsers
2. All browsers
3. Remotes